src/components/GoalPage/GoalPage.tsx

```diff
@@ -158,6 +169,44 @@ export const GoalPage = ({ user, ssrTime, params: { id } }: ExternalPageProps<{
         [goal, invalidateFn, addParticipantMutation],
     );
 
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
why do we have this rule if we disable it without a comment? It looks to me that it's pretty simple to make a manual assertion here